feature_selection/forward_selection.py

```diff
@@ -53,10 +86,9 @@ def forward_selection(scorer, X, y, min_features=1, max_features=5):
         data = {'number': ftr_no_select, 'fn_score':fn_score}
         df = pd.DataFrame(data)
 
-        best_one = np.max(df.fn_score) 
-        if (len(ftr_select) > 0 and best_one < np.max(scores) and len(ftr_select) > min_features):
-            flag = True
-            break            
+        best_one = np.min(df.fn_score) 
+        if (len(ftr_select) > 0 and best_one < np.min(scores) and len(ftr_select) >= min_features):
```

Comment:
Don't stop adding features until best one is higher than min of those taken.
```suggestion
        if (len(ftr_select) > 0 and best_one > np.min(scores) and len(ftr_select) >= min_features):
```